frontend/wallet/src/render/views/onboarding/CompletionStep.re

```diff
@@ -134,58 +132,86 @@ module CopyKeyButton = {
   };
 };
 
+// [@react.component]
```

Comment:
feel free to delete this, we have it stored in git so we can always go back to it if we want to recover it.